test/schrodinger/rdkit_extensions/test_cg_coordgen.cpp

```diff
@@ -171,52 +193,54 @@ BOOST_DATA_TEST_CASE(
          {{0, 0},
           {-1.5, -1.5},
           {0, -1.5},
-          {3, -4.5},
-          {3, -3},
-          {1.5, -4.5},
-          {0, -4.5},
-          {0, -3},
-          {-1.5, -4.5}}},
+          {-6, -3},
+          {-6, -4.5},
+          {-4.5, -3},
+          {-3, -3},
+          {-3, -4.5},
+          {-1.5, -3}}},
 
```

Comment:
just making sure: all of the HELM strings with differences in coordinates you included in your description, right?